src/activity-logger/background/pdffingerprint.js

```diff
@@ -0,0 +1,13 @@
+/* eslint eqeqeq: 0 */
+export default function getPDFFingerprint(pdfURL) {
+    const PDFJS = require('../../build/pdf')
+    PDFJS.GlobalWorkerOptions.workerSrc = '../../build/pdf.worker.js'
```

Comment:
A few things about this module:
1. let's make all new modules `.ts` or `.tsx` modules (you can still write your standard JS stuff in those modules if you're new to TS)
2. I'm not sure if you're going to have an issue here with the relative paths. Those are relative to the source code structure although, once the ext is running in the browser post-build, that structure no longer exists and paths are now relative to the produced `extension/` dir. Have a look how we've done it using `extension.getURL` API for our other pdfjs uses: https://github.com/WorldBrain/Memex/blob/develop/src/page-analysis/content_script/extract-pdf-content.js#L9
3. There shouldn't be any reason to instantiate and return a `Promise` here. You can make the overall function `async`, then it implicitly returns a `Promise`. Though that assumes the rest of the pdfjs API is promise-based.